halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/validate/v1/canary/google/GoogleCanaryAccountValidator.java

```diff
@@ -40,6 +44,7 @@
   private String halyardVersion;
 
   private Registry registry;
+  private Storage googleCloudStorage;
```

Comment:
How is this field initialized? I assume it should be different for each `GoogleCanaryAccount`, which is passed in `validate`